samples/app-dev/API-Development/REST/allOf_oneOf-anyOf_construct_schema/readme.md

```diff
@@ -72,9 +68,11 @@ Once you have imported both the apps, push the 'AllOf_OneOf_AnyOf_Keywords_APISp
 Now push the 'Invoke_AllOf_OneOf_AnyOf_Keywords_APISpec_Service' app and scale the app to 1. Go to 'Environment Controls' tab -> 'Application Variables' and edit the default value of the 'InvokeRestURL' application property to point to the endpoint URL of the producer Rest service app.
 ![Application property on Endpoint tab](./import-screenshots/updateServiceUrl_EnvControlstab.png)
 
-To run the app in Flogo Enterprise, create appropriate binaries for both the apps and run the 'AllOf_OneOf_AnyOf_Keywords_APISpec_Service'. Export the URL of the Service app in the 'Invoke_AllOf_OneOf_AnyOf_Keywords_APISpec_Service' app before running the invoking app like this:
+To run the app binary, create appropriate binaries for both the apps and run the 'AllOf_OneOf_AnyOf_Keywords_APISpec_Service'. Export app property and its URL for all the three services in  'Invoke_AllOf_OneOf_AnyOf_Keywords_APISpec_Service' app before running the invoking app like this:
 
- $ export InvokeRestURL="http://localhost:9998"
+ $ export allOf_Service="https://1a91e01153ca0b9e0ff24561c8-integration.local.cic2apps.pro/nvetqpftntbygh233hxmhvvwcn2sdesb/allOfAPISpecRestTrigger"
+ $ export anyOf_Service="https://1a91e01153ca0b9e0ff24561c8-integration.local.cic2apps.pro/nvetqpftntbygh233hxmhvvwcn2sdesb/anyOfAPISpecRestTrigger"
+ $ export oneOf_Service="https://1a91e01153ca0b9e0ff24561c8-integration.local.cic2apps.pro/nvetqpftntbygh233hxmhvvwcn2sdesb/oneOfAPISpecRestTrigger"
```

Comment:
Since service binary is running on local, it can't have TCI URLs
Pls mention localhost URLs with each port on which respective service is running.